ONSdigital/florence
# Encrypt florence uploads

### What

Encrypt files which are written to S3 by default
 
### How to review

- Create a RSA public/private key pair and generate private RSA PEM file in your $HOME
- Read the file contents into an environment variable: e.g if the file is called private.pem, then set:
`export RSA_PRIVATE_KEY=$(cat $HOME/private.pem)`
- Start florence on this branch
- Upload a new dataset (instance) using florence
- Verify that the uploaded file is encrypted

***NOTE THIS SHOULD NOT BE MERGED UNTIL THE BACKEND WORK HAS BEEN DONE TO DECRYPT FILES***

### Who can review

Anyone
